docs/components/clients/Clients.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { users } from "./users";
+import { Logo } from "./Logo";
+
+export function Clients({ linked }: { linked?: boolean }) {
+  const showcaseDark = [];
+  const showcaseLight = [];
+  users.forEach((user, idx) => {
```

Comment:
- `idx` isn't used. do we have lint against unused declarations?
- `for...of` would be nice!

Reply:
Ah good catch on the idx, will remove since we don't need that for the key. 

[for...of](https://caniuse.com/?search=for...of) actually has sub par browser support compared with the array methods.

Reply:
Don't we transpile anyways?

Reply:
We do, I would still argue forEach is easier to read, and more conventional in this case.

Reply:
i'm all about the `for...of` these days :D, but yeah, for most intents and purposes, it's a stylistic issue.